### PR TITLE
[TASK] Introduce TYPO3 CS for PHP

### PR DESCRIPTION
**Pull request by linawolf**
I think we should also run this in the Github Actions but I am not sure how

### CODE REVIEW

**Comment by brotkrueml** [comment]
> I think we should also run this in the Github Actions but I am not sure how

Will have a look shortly.

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+$config = \TYPO3\CodingStandards\CsFixerConfig::create();
+$config
+    ->getFinder()->in(__DIR__)
+    ->exclude([
+        'Documentation',
```
Comment (brotkrueml) [inline .php-cs-fixer.dist.php:7]: What is the difference of the both directories? "Documentation" is not available yet.

Comment (linawolf) [reply .php-cs-fixer.dist.php:7]: I always put the stuff I want to test into Documentation

Comment (brotkrueml) [reply .php-cs-fixer.dist.php:7]: Then, should be the stuff from "Documentation" be code-styled?

And what is the purpose of "docs"? Actually, there is only one rst file. Should it be delivered to the users?

Comment (linawolf) [reply .php-cs-fixer.dist.php:7]: The docs was just added by @jaapio and explains how you can use the mono-repository. So I guess it should be delivered to the user if the complete reposiory is delivered to them?

Comment (linawolf) [reply .php-cs-fixer.dist.php:7]: And no, I wouldn't want an external script to code-style my test Data as it comes from a different source, might have different PHP versions etc

Comment (brotkrueml) [reply .php-cs-fixer.dist.php:7]: Aaaah, it is an "exclude", sorry, just looked at the folder names....

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**